js/blocks.js

```diff
@@ -7066,6 +7089,13 @@ function Blocks(activity) {
             this.addDefaultBlock(parentBlock, thisBlock);
         }
 
+        if (myBlock.name == "everybeatdonew"){
+            let turtleID = myBlock.turtleID ;
+            let tur;
+            for(tur of turtles.getTurtleList())if (turtleID == tur.id)break;
+            this.sendStackToTrash(tur.startBlock);
+        }
```

Comment:
There are a few leftover tasks in `Turtles`, however, `turtleList` has been updated to the new `getter` syntax, so access it like: `turtles.turleList`.

Also, I think it's better to use `{` `}` here.